### ISSUE TITLE
Wrong volume mapping in keycloak compose file

### ISSUE THREAD

**Issue by marc-b2** [AOT-Technologies/forms-flow-ai#1200]
### Please provide information about your setup

- formsflow.ai Version: 5.1.0

### Steps to Reproduce

1. follow documentation for docker full deployment

### Observed Behaviour

Changes in keyloak database are not stored persistantly.

### Expected Behaviour

Changes in keycloak database should be stored persistantly.

### Fix

volume mapping for `keycloak-db` in `forms-flow-idm/keycloak/docker-compose.yml` should look like this

`./postgres/keycloak:/var/lib/postgresql/data`

as provided [here](https://github.com/docker-library/docs/tree/master/postgres#where-to-store-data)

**Comment by sumesh-aot** [comment]
Thanks @marc-b2 for raising this issue.
We have 2 PRs in place to fix this in local development (https://github.com/AOT-Technologies/forms-flow-ai/pull/1208) and in deployment (https://github.com/AOT-Technologies/forms-flow-ai-deployment/pull/37/files)

**Comment by sreehari-aot** [comment]
is this issue specific to mac only? @sumesh-aot @marc-b2

**Comment by marc-b2** [comment]
@sreehari-aot no, this issue is not specific to the host system. it is about the path where the data is stored inside the postgres container.
